lab_3_keywords_textrank/main.py

```diff
@@ -109,7 +119,10 @@ def __init__(self) -> None:
         """
         Constructs all the necessary attributes for the text encoder object
         """
-        pass
+        # self._word2id = {str: int}
```

Comment:
what is it?

Reply:
i thought this would work, but it turned out to have errors in check styles, so i deleted it